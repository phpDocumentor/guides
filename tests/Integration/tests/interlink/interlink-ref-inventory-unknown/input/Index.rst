
..  _doc:

==============
Document Title
==============

Lorem Ipsum Dolor.

See the :ref:`TYPO3 documentation <unknowninventory:doc>`!
