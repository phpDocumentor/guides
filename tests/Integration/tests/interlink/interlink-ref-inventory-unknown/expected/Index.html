<!-- content start -->
    <div class="section" id="document-title">
            <a id="doc"></a>
            <h1>Document Title</h1>

    <p>Lorem Ipsum Dolor.</p>


    <p>See the TYPO3 documentation!</p>

    </div>
<!-- content end -->
